app/modules/phonebook/models_phonebook.py

```diff
@@ -20,20 +29,37 @@ class Membership(Base):
         primary_key=True,
     )
     mandate_year: Mapped[int] = mapped_column(Integer, primary_key=True)
-    role_name: Mapped[str] = mapped_column(String, nullable=False)
-    role_tags: Mapped[str] = mapped_column(String, nullable=False)
+    role_name: Mapped[str]
+    role_tags: Mapped[str]
+    member_order: Mapped[int]
 
 
 class Association(Base):
     __tablename__ = "phonebook_association"
 
-    id: Mapped[str] = mapped_column(
+    id: Mapped[str] = mapped_column(String, primary_key=True, index=True)
+    name: Mapped[str] = mapped_column(String, index=True)
+    description: Mapped[str] = mapped_column(String, nullable=True)
+    kind: Mapped[Kinds]
+    mandate_year: Mapped[int]
+    deactivated: Mapped[bool]
+    associated_groups: Mapped[list["CoreGroup"]] = relationship(
+        "CoreGroup",
+        secondary="phonebook_association_associated_groups",
+        lazy="selectin",
```

Comment:
Is there a benefit to the use of "selectin" instead of "joined"?

Reply:
I just copied the way it was done in reception's PR